src/rpc/rawtransaction.cpp

```diff
@@ -211,7 +211,7 @@ UniValue gettxoutproof(const UniValue& params, bool fHelp)
             "\nNOTE: By default this function only works sometimes. This is when there is an\n"
             "unspent output in the utxo for this transaction. To make it always work,\n"
             "you need to maintain a transaction index, using the -txindex command line option or\n"
-            "specify the block in which the transaction is included in manually (by blockhash).\n"
+            "specify the block in which the transaction is included manually (by blockhash).\n"
```

Comment:
> NOTE: By default this function only works sometimes. This is when there is an unspent output in the utxo for this transaction. To make it always work, you need to maintain a transaction index, using the -txindex command line option or specify the block in which the transaction is included in manually (by blockhash).

I think this is what the note is trying to say:

This function only works when the transaction(s) can be located. Transactions can be located if any of these conditions are met:

1) the transaction has at least one unspent output
2) the transaction index is active (enabled using the -txindex command line option)
3) the optional blockhash parameter is provided and the corresponding block contains the transaction

Aside: why does the function accept multiple txids but only one blockhash? It appears that for this function to work all the transactions have to be in the same block. In which case it should be possible to find their containing block if any one of them has an unspent output. But the code appears to only attempt to look up the last specified txid. But this is way outside the scope of this typo-fixing issue!